plsql/PlSqlParser.g4

```diff
@@ -65,6 +67,8 @@ unit_statement
     | comment_on_table
 
     | anonymous_block
+
+    | grant_statement
```

Comment:
This grammar also has `GRANT ALL? swallow_to_semi` alternative for `statement` rule. Is it duplicate and should be removed?